mantidproject/mantid
# Toml v1 feedback

**Description of work.**

*We're not at the point where the group is comfortable committing completely to V1 yet, so we will leave version as V0 (despite having breaking changes during this beta testing phase), since there is a limited number of files we've converted.*

List of changes:
- Fixed TomlValidationErrors causing a Sad Mantid
- List of changes to TOML schema as documented at the top of `Sans-Toml-v1.rst`

@smk78 

**To test:**
- Download the updated [training test data](https://sourceforge.net/projects/mantid/files/Sample%20Data/TrainingCourseData.zip/download)
- Find the `loqdemo` folder inside
- Open ISIS SANS 
- Set Mask file to the **.toml** file inside
- Set Batch file to the .csv inside
- Run the reduction, it should complete successfully 

Fixes #30514 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
